src/main/java/com/serenitydojo/Dog.java

```diff
@@ -0,0 +1,34 @@
+package com.serenitydojo;
+
+public class Dog {
+    private String name;
+    private String favoriteToy;
+    private int age;
+    private boolean isFed;
+    public static final String DOG_NOISE = "WOOF WOOF!!";
+
+    public Dog(String name, String favoriteToy, int age) {
+        this.name = name;
+        this.favoriteToy = favoriteToy;
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+    public String getFavoriteToy() {
+        return favoriteToy;
+    }
+    public int getAge() {
+        return age;
+    }
+    public void makeNoise() {
+        System.out.println("Woof");
```

Comment:
To test this you might want to have the method return a String and return "Woof", so you can test that - testing what is written to the console is trickier.